lib/tablewriter/tablewriter.go

```diff
@@ -0,0 +1,129 @@
+package tablewriter
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/acarl005/stripansi"
+)
+
+type Column struct {
+	Name         string
+	SeparateLine bool
+}
+
+type TableWriter struct {
+	cols []Column
+	rows []map[int]string
+}
+
+func Col(name string) Column {
+	return Column{
+		Name:         name,
+		SeparateLine: false,
+	}
+}
+
+func NewLineCol(name string) Column {
+	return Column{
+		Name:         name,
+		SeparateLine: true,
+	}
+}
+
+// Unlike text/tabwriter, this works with CLI escape codes, and allows for info
+//  in separate lines
+func New(cols ...Column) *TableWriter {
+	return &TableWriter{
+		cols: cols,
+	}
+}
+
+func (w *TableWriter) Write(r map[string]interface{}) {
+	// this can cause columns to be out of order, but will at least work
```

Comment:
we should sort this, out of order columns would be pretty obnoxious UX

Reply:
This only becomes out of order if the columns aren't defined in the constructor